app/src/main/res/layout/activity_main.xml

```diff
@@ -4,17 +4,242 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="#00BCD4"
+    android:backgroundTint="#00BCD4"
+    android:foregroundTint="#FF9800"
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/orangeTextView"
+        android:layout_width="137dp"
```

Comment:
Please consider avoiding to add hardcoded values in the layouts like dimensions, colors, strings. Define them in the dedicated files.